src/shared/components/case-viewer/case-viewer.component.ts

```diff
@@ -8,19 +8,19 @@ import { Observable } from 'rxjs';
 import { Subscription } from 'rxjs/Subscription';
 import { CaseField } from '../../domain/definition';
 import { ShowCondition } from '../../directives/conditional-show/domain';
-import { Draft } from '../../domain';
+import { Draft, DRAFT_QUERY_PARAM } from '../../domain';
 import { HttpError } from '../../domain/http';
 import { OrderService } from '../../services/order';
 import { CaseView, CaseViewTrigger } from '../../domain/case-view';
 import { DeleteOrCancelDialogComponent } from '../../components/dialogs';
-import { DRAFT_QUERY_PARAM } from '../../domain';
 import { AlertService } from '../../services/alert';
 import { CallbackErrorsContext } from '../../components/error/domain';
 import { DraftService } from '../../services/draft';
 import { MatDialog, MatDialogConfig } from '@angular/material';
 import { CaseService } from '../case-editor';
-import { NgZone } from '@angular/core';
+import { Type } from 'class-transformer';
 
+// @dynamic
```

Comment:
why this comment? what's the meaning?

Reply:
Its to get away from errors during compilation around lambda expression